db/migrate/20171105174415_add_military_status_to_veterans.rb

```diff
@@ -0,0 +1,5 @@
+class AddMilitaryStatusToVeterans < ActiveRecord::Migration[5.1]
+  def change
+    add_column :veterans, :military_status, :integer
+  end
+end
```

Comment:
can we rollback this migration specifically because we have it defined already as "roles"